oggm/utils/_downloads.py

```diff
@@ -2190,6 +2202,9 @@ def default_dem_source(rgi_id):
     sel = cfg.DEM_SOURCE_TABLE[rgi_reg].loc[rgi_id]
     for s in ['NASADEM', 'COPDEM', 'GIMP', 'REMA', 'TANDEM', 'MAPZEN']:
         if sel.loc[s] > 0.75:
+            # this can go as soon as 'rgi62_dem_frac.h5' is updated:
```

Comment:
@fmaussion what do you think about this? Not sure when/how/where this 'rgi62_dem_frac.h5' gets updated. Or if this might break somewhere else.

Reply:
Yeah I think this is irrelevant for this PR - I really need to rethink this logic (related: https://github.com/OGGM/oggm/issues/1269)